package protoFile.pet.GetBeiJiChengList2313;
import "protoFile/pet/petInfo.proto";
message GetBeiJiChengListRequest{
     required int32 id=1;//角色id
     required int32 petId=2;//宠物id
}
message GetBeiJiChengListResponse{
     optional bool result=1;
     optional string message=2;
     repeated PetInfoList petInfoList = 3;//宠物信息列表
}
message PetInfoList{
     optional int32 reqCoin=1;//花费金币
     optional PetInfo info = 2;//宠物信息
}